File: fixtures/rust_output/get_with_form.rs
extern crate reqwest;
use reqwest::multipart;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let form = multipart::Form::new()
        .text("from", "test@tester.com")
        .text("to", "devs@tester.net")
        .text("subject", "Hello")
        .text("text", "Testing the converter!");

    let res = reqwest::Client::new()
        .post("https://api.net/v3")
        .basic_auth("test", Some(""))
        .multipart(form)
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}

File: test/fixtures/javascript-xhr/get_with_form.js
const form = new FormData();
form.append('from', 'test@tester.com');
form.append('to', 'devs@tester.net');
form.append('subject', 'Hello');
form.append('text', 'Testing the converter!');

let xhr = new XMLHttpRequest();
xhr.withCredentials = true;
xhr.open(
  'POST',
  'http://localhost:28139/v3',
  true,
  'test',
  ''
);

xhr.onload = function() {
  console.log(xhr.response);
};

xhr.send(form);
